src/test/java/com/epam/test_generator/services/ProjectServiceTest.java

```diff
@@ -0,0 +1,398 @@
+    private UserDTO simpleUserDTO2;
+    private Set<UserDTO> userDTOSet;
+    private ProjectFullDTO simpleProjectFullDTO1;
+
+    private static final long SIMPLE_PROJECT_ID = 1L;
+    private static final long SIMPLE_SUIT_ID = 2L;
+    private static final long SIMPLE_USER_ID = 3L;
+
+    @Before
+    public void setUp() throws Exception {
+        simpleUser1 = new User(
+            "testUser1@mail.com",
+            "testPassword1",
+            new Role("GUEST")
+        );
+        simpleUser1.setId(1L);
+        simpleUser2 = new User(
+            "testUser2@mail.com",
+            "testPassword2",
+            new Role("GUEST")
+        );
+        simpleUser2.setId(2L);
+        simpleUserDTO1 = new UserDTO(
+            "testPassword1",
+            "GUEST",
+            "testUser1@mail.com"
+        );
+        simpleUserDTO1.setId(1L);
+        simpleUserDTO2 = new UserDTO(
+            "testPassword2",
+            "GUEST",
+            "testUser2@mail.com"
+        );
+        simpleUserDTO2.setId(2L);
+        simpleProject1 = new Project(
+            "project1",
+            "project1",
+            null,
+            userSet,
+            true
+        );
+        simpleProject1.setId(SIMPLE_PROJECT_ID);
+
+        simpleProject2 = new Project(
+            "project2",
+            "project2",
+            null,
+            userSet,
+            true
+        );
+        simpleProject2.setId(2L);
+        simpleProject1DTO = new ProjectDTO(
+            "project1",
+            "project1",
+            true,
+            userDTOSet
+        );
+        simpleProject1DTO.setId(2L);
+        simpleProject2DTO = new ProjectDTO(
+            "project2",
+            "project2",
+            true,
+            userDTOSet
+        );
+        simpleProject2DTO.setId(2L);
+        simpleProjectFullDTO1 = new ProjectFullDTO(
+            "project1",
+            "project1",
+            null,
+            userDTOSet,
+            true
+        );
+        simpleProjectFullDTO1.setId(1L);
+
+        userSet = Stream.of(simpleUser1, simpleUser2)
+            .collect(Collectors.toSet());
+        simpleProject1.setUsers(userSet);
+        simpleProject2.setUsers(userSet);
+        userDTOSet = Stream.of(simpleUserDTO1, simpleUserDTO2)
+            .collect(Collectors.toSet());
+        simpleProject1DTO.setUsers(userDTOSet);
+        simpleProject2DTO.setUsers(userDTOSet);
+        simpleProjectFullDTO1.setUsers(userDTOSet);
+        expectedProjectDTOs = Stream.of(simpleProject1DTO, simpleProject2DTO)
+            .collect(Collectors.toList());
+        expectedProjects = Stream.of(simpleProject1, simpleProject2)
+            .collect(Collectors.toList());
+    }
+
+    @Test
+    public void getProjects_expectedProjectsList() {
+        when(projectDAO.findAll()).thenReturn(expectedProjects);
+        final List<ProjectDTO> projects = projectService.getProjects();
+        assertEquals(projects, projects);
```

Comment:
This assertion never fail

Reply:
fixed (your message about "expected params" above)